worldedit-core/src/main/java/com/boydti/fawe/beta/implementation/blocks/CharBlocks.java

```diff
@@ -16,6 +16,13 @@
     protected static final Section FULL = new Section() {
         @Override
         public final char[] get(CharBlocks blocks, int layer) {
+            if (blocks.blocks[layer] == null) {
+                if (blocks.sections[layer].isFull()) {
+                    throw new IllegalStateException("Array cannot be null: " + blocks.getClass());
```

Comment:
I cherry-picked your commits on the feature/tuinity-starlight branch and I throws this exception now when doing `//fillr green_wool 200`.
Stacktrace: 
https://paste.gg/p/anonymous/d32d21c5521b4545becbf0dd91ab3f4b